From db875e781356aa9baab7bbdc7a8e2ee11aa5bbb1 Mon Sep 17 00:00:00 2001
From: Nikoloz Adeishvili <Nikoloz_Adeishvili@epam.com>
Date: Tue, 12 Jul 2022 01:45:46 +0400
Subject: [PATCH 2/3] Change last two lines in firstFile.txt from git_1

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index f00c965..4da93ef 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -6,5 +6,5 @@
 6
 7
 8
-9
-10
+19
+20
-- 
2.36.1.windows.1

