From 264b49bd33ac700e3c374bc8ba36a8e9807f1dc8 Mon Sep 17 00:00:00 2001
From: Nikoloz Adeishvili <Nikoloz_Adeishvili@epam.com>
Date: Tue, 12 Jul 2022 01:46:29 +0400
Subject: [PATCH 3/3] Change first two lines in firstFile,txt from git_1

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index 4da93ef..6181477 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-1
-2
+11
+12
 3
 4
 5
-- 
2.36.1.windows.1

